diff --git a/tests/manage/z_cluster/nodes/test_nodes_maintenance.py b/tests/manage/z_cluster/nodes/test_nodes_maintenance.py
index 94babae72..094a6d8ba 100644
--- a/tests/manage/z_cluster/nodes/test_nodes_maintenance.py
+++ b/tests/manage/z_cluster/nodes/test_nodes_maintenance.py
@@ -27,7 +27,7 @@ from ocs_ci.ocs.node import (
 )
 from ocs_ci.ocs.cluster import validate_existence_of_blocking_pdb
 from ocs_ci.framework import config
-from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.pytest_customization.marks import brown_squad, skipif_ibm_power
 from ocs_ci.framework.testlib import (
     tier1,
     tier2,
@@ -189,6 +189,7 @@ class TestNodesMaintenance(ManageTest):
     @tier4a
     @skipif_bm
     @skipif_managed_service
+    @skipif_ibm_power
     @pytest.mark.parametrize(
         argnames=["node_type"],
         argvalues=[
@@ -342,6 +343,7 @@ class TestNodesMaintenance(ManageTest):
     @tier4a
     @aws_based_platform_required
     @ipi_deployment_required
+    @skipif_ibm_power
     @pytest.mark.parametrize(
         argnames=["interface"],
         argvalues=[
@@ -461,6 +463,7 @@ class TestNodesMaintenance(ManageTest):
     @bugzilla("1861104")
     @bugzilla("1946573")
     @skipif_managed_service
+    @skipif_ibm_power
     @skipif_more_than_three_workers
     @pytest.mark.polarion_id("OCS-2524")
     @tier4a
