diff --git a/ocs_ci/ocs/node.py b/ocs_ci/ocs/node.py
index e7072c0db..b0990b43b 100644
--- a/ocs_ci/ocs/node.py
+++ b/ocs_ci/ocs/node.py
@@ -282,7 +282,7 @@ def drain_nodes(node_names, timeout=1800, disable_eviction=False):
         else:
             ocp.exec_oc_cmd(
                 f"adm drain {node_names_str} --force=true --ignore-daemonsets "
-                f"{drain_deletion_flag}",
+                f"{drain_deletion_flag} --timeout=1810s",
                 timeout=timeout,
             )
     except TimeoutExpired:
